openc3/spec/interfaces/http_server_interface_hal_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'spec_helper'
```

Comment:
What is the `hal` in the file name for? We already have a `http_server_interface_spec.rb`. These additional methods should just go in there.

Reply:
WIP.  The hal file is gone.